Assets/LeapMotion/Core/Scripts/LeapXRServiceProvider.cs

```diff
@@ -8,7 +8,11 @@
  ******************************************************************************/
 
 using UnityEngine;
+#if UNITY_2017_2_OR_NEWER
```

Comment:
We should avoid having any #ifdefs to support XR/VR across Unity versions outside of the dedicated file for it: XRSupportUtil. So no script other than it should need `using UnityEngine.XR` or use an XR/VR namespace.

I was holding off on adding support for prior Unity versions to `develop` so that we could make all of the changes at once before a release cycle goes through, but if we're going to do it in this PR, we should do it in a way that doesn't pollute our code with #ifdefs all over. :)